apps/chargent/boot.js

```diff
@@ -27,6 +27,7 @@
             // TODO ? customizable
             Bangle.buzz(500);
             setTimeout(() => Bangle.buzz(500), 1000);
+            E.showAlert("Charge peak reached").then(() => load());
```

Comment:
I believe `load()` is needed, as otherwise the alert draws over a clock face and when dismissed, leaves a blank screen - although this will bump the user out of any app they're in